### PR TITLE
Implemented default channel changes

### PR DESCRIPTION
**Pull request by Almighty-Alpaca**
See hammerandchisel/discord-api-docs#329

### CODE REVIEW

**Comment by kantenkugel** [comment]
Looks fine for me

**Comment by MinnDevelopment** [comment]
Is my comment going to get any response?

**Comment by kantenkugel** [comment]
would call it getFirstReadableChannel() in guild level

File: src/main/java/net/dv8tion/jda/core/entities/Guild.java [hunk 1]
```diff
@@ -573,11 +574,33 @@
...
      * @return The {@link net.dv8tion.jda.core.entities.TextChannel TextChannel} representing the public channel for this guild.
+     *
+     * @see    net.dv8tion.jda.core.entities.Member#getDefaultChannel()
      */
+    @Deprecated
     TextChannel getPublicChannel();
 
     /**
+     * The default {@link net.dv8tion.jda.core.entities.TextChannel TextChannel} for a {@link net.dv8tion.jda.core.entities.Guild Guild}.
+     * <br>This is the channel that the Discord client will default to opening when a Guild is opened for the first time when accepting an invite.
```
Comment (DV8FromTheWorld) [inline src/main/java/net/dv8tion/jda/core/entities/Guild.java]: This isn't true if the invite is to a TextChannel

**Pull request merged**